WebRoot/messages/ZmMsg_zh_CN.properties

```diff
@@ -2123,6 +2123,18 @@ oAuthAppDevice = \u8bbe\u5907
 oAuthAppApprovedDate = \u5df2\u6279\u51c6
 removeOAuthAppAuthorization = \u79fb\u9664 App \u9a8c\u8bc1
 oAuthAppAuthorizationRemoveConfirm = \u6b64\u4e3e\u5c06\u4f1a\u5728\u60a8\u7684 "{1}" \u4e0a\u79fb\u9664\u5bf9 "{0}" \u7684\u9a8c\u8bc1\u3002
+mobileConFigurationDownloadLabel = \u4e0b\u8f7d
+mobileConfiguration = \u79fb\u52a8\u914d\u7f6e
+mobileConfigurationAll = \u7535\u5b50\u90ae\u4ef6\uff0c\u65e5\u5386\u548c\u4efb\u52a1\uff0c\u8054\u7cfb\u4eba
+mobileConfigurationCaldav =	\u65e5\u5386\u548c\u4efb\u52a1
+mobileConfigurationCarddav = \u5f80\u6765
+mobileConfigurationDav = \u65e5\u5386\u548c\u4efb\u52a1\uff0c\u8054\u7cfb\u4eba
+mobileConfigurationDownload = \u60a8\u53ef\u4ee5\u4e0b\u8f7d\u006d\u006f\u0062\u0069\u006c\u0065\u0063\u006f\u006e\u0066\u0069\u0067\u914d\u7f6e\u6587\u4ef6\u4ee5\u81ea\u52a8\u914d\u7f6e\u652f\u6301\u5b83\u7684\u8bbe\u5907\u548c\u5e94\u7528\u7a0b\u5e8f\u3002
+mobileConfigurationDownloadList = \u914d\u7f6e\uff1a
+mobileConfigurationImap = \u7535\u5b50\u90ae\u4ef6
+mobileConfigurationInstruction = \u0049\u004d\u0041\u0050\uff0c\u0043\u0061\u006c\u0044\u0041\u0056\u548c\u0043\u0061\u0072\u0064\u0044\u0041\u0056\u662f\u5f00\u653e\u534f\u8bae\uff0c\u7528\u4e8e\u5c06\u60a8\u7684\u5e10\u6237\u8d44\u6e90\u4e0e\u79fb\u52a8\u8bbe\u5907\u6216\u684c\u9762\u5e94\u7528\u7a0b\u5e8f\u540c\u6b65\u3002
+mobileConfigurationOptions = <div><p>\uff06bull;\uff06ensp; \u0049\u004d\u0041\u0050 \u0066\u006f\u0072 \u0045\u006d\u0061\u0069\u006c</p><p>\uff06bull;\uff06ensp; \u0043\u0061\u006c\u0044\u0041\u0056 \u0066\u006f\u0072 \u0043\u0061\u006c\u0065\u006e\u0064\u0061\u0072 \u0061\u006e\u0064 \u0054\u0061\u0073\u006b\u0073</p><p>\uff06bull;\uff06ensp; \u0043\u0061\u0072\u0064\u0044\u0041\u0056 \u0066\u006f\u0072 \u0043\u006f\u006e\u0074\u0061\u0063\u0074\u0073</p></div>
```

Comment:
dont convert <div><p>&bull;&ensp;

Reply:
Done.

Reply:
`<p>\uff06bull;\uff06ensp;` 
convert to 

`<p>&bull;&ensp;`

Reply:
Its already there.